lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_for_non_romanians.govspeak.erb

```diff
@@ -39,13 +39,7 @@
     <%= render partial: 'partner_naturalisation_in_uk.govspeak.erb' %>
   <% end %>
 
-  <%= render partial: 'consular_fees_table.govspeak.erb',
-             locals: { calculator: calculator } %>
-
-  <%= render partial: 'link_to_consular_fees.govspeak.erb',
-             locals: { calculator: calculator } %>
-
-  <%= render partial: 'payment_information.govspeak.erb',
+  <%= render partial: 'os_affirmation/fees.govspeak.erb',
```

Comment:
Minor: I was initially confused by the name of the outcome - it wasn't obvious to be that it was specific to Romania. However, improving the name of the outcome is probably out-of-scope for this PR.